domains/cooper.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Domain for my Portfolio Thing",
+  "repo": "https://github.com/cooper25/portfolio2021",
+  "owner": {
+    "username": "cooper25",
+    "email": "cooper@tsor.is"
+  },
+  "record": {
+    "URL": "https://cppr.dev"
```

Comment:
hi there, do you actually want a url redirect? if you want a masked website, this should be a cname. please confirm you actually want a url redirect, or ask if you are unsure.